packages/cactus-plugin-ledger-connector-daml/src/test/typescript/integration/daml-get-transaction.test.ts

```diff
@@ -0,0 +1,232 @@
+import { v4 as uuidv4 } from "uuid";
+import "jest-extended";
+
+import { DamlTestLedger } from "@hyperledger/cactus-test-tooling";
+
+import { PluginLedgerConnectorDAML } from "../../../main/typescript/plugin-ledger-connector-daml";
+
+import { DefaultApi } from "../../../main/typescript/generated/openapi/typescript-axios/index";
+
+import {
+  IListenOptions,
+  LogLevelDesc,
+  LoggerProvider,
+  Logger,
+  Servers,
+} from "@hyperledger/cactus-common";
+
+import { pruneDockerAllIfGithubAction } from "../../../../../../packages/cactus-test-tooling/src/main/typescript/github-actions/prune-docker-all-if-github-action";
```

Comment:
@raynatopedrajeta Please use the package names when importing code from other packages. This sort of relative path jump across packages will break the build in unexpected ways.